CONTRIBUTING.md

```diff
@@ -15,38 +15,44 @@ Can you read this? Then you! Simply consult the short [style guidelines](#style-
 
 1. <a href="https://github.com/NREL/HPC/fork">Fork this repo <img src="https://img.shields.io/github/forks/NREL/HPC.svg?style=social"></a> 
-2. _(optional)_ To edit locally on your machine, do either of:
+1. _(optional)_ To edit locally on your machine, do either of:
    * `git clone https://github.com/`\<your GitHub username\>`/HPC` for only the base repo.
-   * `git clone https://github.com/`\<your GitHub username\>`/HPC.wiki` to clone and edit the [Wiki](https://github.com/NREL/HPC/wiki) if you intend to have highly verbose documetation.
-     
-     ***Note that there does not have to be an entry for your content in both repos.** If you only want to construct explanitory documents, they do not need a directory in the base repo. Similarly, example or utility scripts that don't need a lot of explanation don't need an entry in the [Wiki](https://github.com/NREL/HPC/wiki). For more info on the intended purpose of each repo, see the [style guidelines](#style-guidelines) below.*
-3. Change something (_after_ consulting the [style guidelines](#style-guidelines))
-4. `git add` the change(s)
-5. `git commit` with a useful commit message!
-6. `git push`
-7. Make a pull-request (shiny green button on your fork of the repo's GitHub webpage.)
+1. If contributing to gh-pages:
+   * `git checkout gh-pages`
+1. Change something (_after_ consulting the [style guidelines](#style-guidelines))
+1. `git add` the change(s)
+1. `git commit` with a useful commit message!
+1. `git push`
+1. Make a pull-request (shiny green button on your fork of the repo's GitHub webpage.)
 
 Alternatively, you may send your contributions via e-mail attachment to HPC-Help@nrel.gov with subject of "NREL HPC GitHub Contribution" and the body detailing what changes you made (the more specific the better).
 
 ### Why should I contribute?
-Something something good Samaritan, benefit the community, searchable knowledge-base, etc. 
+Benefit the community, searchable knowledge-base, etc. 
 
 ---
 
 ## Style Guidelines
 
->### **TL;DR&mdash;this repository should be predominantly composed of scripts/source code/executables or other things that _do_ something on the HPC systems, and the Wiki should predominantly be for explanations about what/how/why.**
->### That said, still getting familiar with the details below will improve the chance your contributions are submitted in a a readily-acceptable format.
-
 ### ***New to Markdown?***
-Appropriately enough, we have a documentation for that! Simply start with the [README in the Markdown module](/markdown/README.md). Not to mention, the raw-contents of any `.md` file can be used as a reference for how to style content a certain way.
+Appropriately enough, we have a documentation for that! Simply start with the [README in the Markdown module](/general/beginner/markdown/README.md). Not to mention, the raw-contents of any `.md` file can be used as a reference for how to style content a certain way.
```

Comment:
Appropriately enough, we have documentation for that! Simply start with the [README in the Markdown module](/general/beginner/markdown/README.md). The raw contents of any `.md` file can be also used as a reference for how to style content a certain way.